librz/core/cmd_open.c

```diff
@@ -538,8 +538,9 @@ static void rz_core_cmd_omt(RzCore *core, const char *arg) {
 		rz_table_add_rowf(t, "ddxxxxxss", m->id, m->fd, pa, pa_end, pa_size, va, va_end, rz_str_rwx_i(m->perm), name);
 	}
 
+	t->showFancy = true;
 	if (rz_table_query(t, arg)) {
-		char *ts = rz_table_tofancystring(t);
+		char *ts = rz_table_tostring(t);
```

Comment:
what's the difference between `rz_table_tofancystring` and `rz_table_tostring`

Reply:
https://github.com/rizinorg/rizin/blob/5b95327c17699e787a58b9928d6b4a9101f31fd6/librz/util/table.c#L408-L412
`rz_table_tostring` calls `rz_table_tofancystring` when needed as seen above so it's actually 1 level higher

Reply:
ok.